src/pages/MonitorLogEditPage.jsx

```diff
@@ -16,12 +30,16 @@ import { FormProvider, useForm } from 'react-hook-form';
 import { FiArrowUp, FiCheck } from 'react-icons/fi';
 // import { OCHBackend } from '../common/utils';
 import LogTemplateSwitcher from '../components/LogTemplateSwitcher';
+import NewQuestionModal from '../components/NewQuestionModal';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'NewQuestionModal' is defined but never used.